packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -305,67 +305,47 @@ export async function scanParentDirs(
 ): Promise<ScanParentDirsResult> {
   assert(path.isAbsolute(destPath));
 
-  let cliType: CliType = 'yarn';
-  let packageJson: PackageJson | undefined;
-  let packageJsonPath: string | undefined;
-  let currentDestPath = destPath;
+  const pkgJsonPath = await walkParentDirs({
+    base: '/',
```

Comment:
Maybe let's add a TODO comment here to allow `base` to be specified as a parameter? I'm also not sure if this will work on Windows.

Reply:
I guess we'll find out when the unit tests run 😄 

Ideally I would like to pass in the `repoRootPath` here but `scanParentDirs` doesn't accept that parameter today, so we at least need a fallback